locales/en/data-classes.ftl

```diff
@@ -66,6 +66,7 @@ instant-messenger-identities = Instant messenger identities
 ip-addresses = IP addresses
 job-applications = Job applications
 job-titles = Job titles
+licence-plates = Licence Plates
```

Comment:
I know it's silly, but can we add a note that these reference vehicles? Also, what's up with the British spelling?

Reply:
Added a note. Ha! I just copy-pasted the strong from the JSON file. I think Troy is Australian, so he probably uses the British version (same as Canada). Good catch!